### ISSUE TITLE
GH Action doesn't populate outputs

### ISSUE THREAD

**Issue by JonasKruckenberg** [jbolda/covector#215]
While working on a GitHub workflow that needed to know which packages have been published I noticed that the covector action is not actually populating most of the outputs. 
When I log the outputs after the action ran I get this for the `version` command:

```
status: 
willPublish: 
commandRan: version
successfulPublish: false
packagesPublished: 
```
And this for the `publish` command:
```
status: 
willPublish: 
commandRan: publish
successfulPublish: true
packagesPublished: 
```
(Note: I omitted the change output, it works fine! I also ensured that a package was actually released the during publish)

**Comment by jbolda** [comment]
Some of these were added/fixed in #211 and will be published with #212. You may be able to try using the release branch for the moment until it is published.

This is where I was first making real use of it: https://github.com/jbolda/netlify_deploy/blob/d1be9bc5f292f9297cf672b0a0704266b8519cd7/.github/workflows/covector-version-or-publish.yml

**Comment by jbolda** [comment]
I believe we have addressed these issues. Let's reopen if we haven't!
